server/cluster/cluster.go

```diff
@@ -390,25 +400,76 @@ func (c *RaftCluster) checkServices() {
 	}
 }
 
+// checkTSOService checks the TSO service.
+func (c *RaftCluster) checkTSOService() {
```

Comment:
Seems we do not need this wrap function now. We can call `startTSOJobs` directly.

Reply:
for the upcomming PR